mpmp/src/cmds/SellPlot.java

```diff
@@ -0,0 +1,86 @@
+package cmds;
+
+import java.util.Arrays;
+import main.Client;
+import main.Conn;
+import main.ErrCode;
+import model.Player;
+import model.Plot;
+import view.Displayer;
+
+/**
+ * C->S
+ * @author Leander
+ */
+public class SellPlot implements CmdFunc{
+	private Displayer d;
+
+	@Override
+	public void exec(String line, Conn conn) {
+		String[] args = line.split(" ");
+		int price = -1;
+		
+		Player p = Player.search(((Client) conn).getName());
+		if (!p.isPlayer()) {
+			conn.sendErr(ErrCode.NotAPlayer);
+			return;
+		}
+		
+		if (args.length < 4) {
+			conn.sendErr(ErrCode.Usage, "sell-plot <Name des Grundstückes> <Käufer> <Preis>");
```

Comment:
Didn't think of that. Please amend the protocol (increment version number to 2). Also, do it the easy way: plot name, price, buyer. You know where the plot name ends because a number follows, and the player names are at the end, like elsewhere in the proto.